assets/css/main.css

```diff
@@ -2,623 +2,650 @@ html, body {
   margin: 0;
   padding: 0;
   height: 100%;
-  font-family: helvetica, sans-serif;
+  font-family: "mr-eaves-xl-modern", helvetica;
+  font-weight:400;
   line-height: 1.3;
-  color: #2b2b2b;
+  color:#555;
   background: #FBFCFC;
   font-size: 110%;
 }
 
-a {
-  color: #F1853D;
-  text-decoration: none;
+div{
+  width:100%;
 }
 
-button {
-  background: #F1853D;
-  border: 0;
-  color: #fff;
-  padding: 13px 30px;
-  font-size: 1.35em;
+nav ul {
+  list-style: none;
 }
 
-input,textarea {
-  font-size: inherit;
-  font-family: inherit;
-  -webkit-appearance: none;
-  border-radius: 0;
+body {
+  font-size: 95%;
 }
 
-fieldset {
-  border: 0;
-  padding: 0;
-  margin: 0;
+input, select, textarea{
+  font-family: "mr-eaves-xl-modern", helvetica;
+  font-weight:400;
 }
 
-fieldset[disabled], button[disabled] {
-  opacity: 0.5;
-  filter: alpha(opacity=50);
+h1, h2, h3, h4, h5, h6{
+  font-family: "mr-eaves-xl-modern", helvetica;
+  font-weight:700;
 }
-
-nav ul {
-  list-style: none;
+h1{
+  font-size:64px;
 }
-
-@media screen and (max-width: 650px) {
-  body {
-    font-size: 95%;
-  }
+h2{
+  font-size:48px;
 }
-
-#wrapper {
-  height: 100%;
+h3{
+  font-size:48px;
+color: rgba(24,34,41, 1)
 }
-
-.input-wrapper {
-  margin: 10px 0;
+h4{
+  font-size:24px;
 }
 
-.input-wrapper input, .input-wrapper textarea {
-  width: 100%;
-  border: 1px solid #ccc;
-  padding: 10px;
-  box-sizing: border-box;
+p{
+  font-size:18px;
 }
 
-.input-wrapper textarea {
-  height: 150px;
-}
+/*Top Bar*/
 
-.input-wrapper-secret {
-  display: none;
+.top-bar{
+  padding:20px 0 20px 0;
+  margin:0;
+  width:100%;
+  display:none;
+  position:fixed;
+  top:0;left:0;
+  z-index:998;
+  background:rgba(24,34,41, 1);
```

Comment:
If you aren't using transparency I would just use `rgb()` instead of including the full opacity of 1